### ISSUE TITLE
Compiled to EXE

### ISSUE THREAD

**Issue by xman213** [CSM-BlueRed/Hyperion-web#1]
Can this be compiled to a exe useing pyinstaller?

**Comment by CSM-BlueRed** [comment]
Yes, but if you dont precise imports, an error will be raised and you cant read it cause its an executable, and cant be debug by a terminal. see [this issue](https://github.com/CSM-BlueRed/Impostor/issues/1) before compile.

**Comment by billythegoat356** [comment]
yes it can
